Automattic/delphin
# Framework: Page titles decorator

This is based on @drewblaisdell work in #254,
This adds page titles via a decorator ( issue #79 )
also related to PR #250 .
#### Testing instructions
1. Run `git checkout try/page-titles-decorator` and start your server, or open a [live branch](https://delphin.live/?branch=try/page-titles-decorator)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Navigate the pages and see the title changes
#### Additional notes

**_Check that cold start (served from server) has the correct title**_  
#### Reviews

@Automattic/sdev-feed
